config/locales/en.yml

```diff
@@ -31,3 +31,4 @@ en:
       short: "%m/%d/%Y at %I:%M %p"
   time_ago_in_words: "%{time} ago"
   unauthorized_collection: You do not have access to %{project}.  Please log in or contact the project owner to be added.
+  unsaved_changes_confirmation: You have unsaved changes. Are you sure you want to leave this screen?
```

Comment:
@WillNigel23 this is the new behavior that we're implementing; the tests may not reflect this.

Reply:
Fixed it. We ignored the whole app/assets dir for translations. But now we only exclude anything js, so .js.erb should be checked for translations

Reply:
Oh, that's interesting!  Good catch.